CppCoreGuidelines.md

```diff
@@ -2663,7 +2663,7 @@ Member functions defined in-class are `inline` by default.
 
 ##### Exception
 
-Template functions (incl. template member functions) are normally defined in headers and therefore inline.
+Function templates (incl. member function templates) are normally defined in headers and therefore inline.
```

Comment:
Arguably, this should be something like "(including member functions of class templates and member functions that are templates)".
But either way, this is an improvement.

Reply:
I'll update that.  It did feel a bit fishy, yes.